### PR TITLE
Install bb

### PR DESCRIPTION
**Pull request by ttgenius**
install.py now installs bb after installing cyberhskae
remove install_bb
changed shared.exe to python3
Tested with benchmark AlburyHYP5-1384

### CODE REVIEW

File: scripts/workflow_config.json [hunk 1]
```diff
@@ -3,11 +3,10 @@
   "emod3d_version" : "3.0.4",
   "global_root" : "/nesi/project/nesi00213",
   "bin_process_path" : "/nesi/project/nesi00213/opt/maui/emod3d/3.0.4-gcc/bin/",
-  "qcore_lib_path" : "/nesi/project/nesi00213/opt/maui/qcore",
+  "qcore_lib_path" : "/home/melody.zhu/qcore",
```
Comment (jasonmotha) [inline scripts/workflow_config.json]: Have to reupdate this again. Ideally, let's find a way to stop this being committed in the future.

**Comment by jasonmotha** [comment]
@ttgenius You're going to have to merge master in your branch again and push. There are now merge conflicts.

File: estimation/estimate_wct.py [hunk 2]
```diff
@@ -12,10 +12,10 @@
...
 
 # Better solution for these locations?
-LF_MODEL_DIR = "/nesi/project/nesi00213/estimation/models/LF/"
-HF_MODEL_DIR = "/nesi/project/nesi00213/estimation/models/HF/"
-BB_MODEL_DIR = "/nesi/project/nesi00213/estimation/models/BB/"
-IM_MODEL_DIR = "/nesi/project/nesi00213/estimation/models/IM/"
+LF_MODEL_DIR = "/home/melody.zhu/models/LF/"
+HF_MODEL_DIR = "/home/melody.zhu/models/HF/"
+BB_MODEL_DIR = "/home/melody.zhu/models/BB/"
+IM_MODEL_DIR = "/home/melody.zhu/models/IM/"
```
Comment (jasonmotha) [inline estimation/estimate_wct.py]: Directories still need updating.

File: shared_workflow/shared_defaults.py [hunk 3]
```diff
@@ -14,12 +14,12 @@
 user = getpass.getuser()
 user_root = os.path.join(run_dir, user)  # global_root
 srf_default_dir = os.path.join(global_root, 'RupModel')
-vel_mod_dir = os.path.join(global_root, 'VelocityModels')
+vel_mod_dir = os.path.join(global_root, 'VelocityModel')
 recipe_dir = workflow_config['templates_dir']
 v_mod_1d_dir = os.path.join(global_root, 'VelocityModel', 'Mod-1D')
 gmsa_dir = os.path.join(global_root, 'groundMotionStationAnalysis')
```
[the remark this answers is not in the document]
Comment (ttgenius) [reply shared_workflow/shared_defaults.py]: removed

File: scripts/cybershake/install_cybershake.sh [hunk 4]
```diff
@@ -25,7 +25,7 @@ for line in $fault_list;
 do
     fault=`echo $line | awk '{print $1}'`
     n_rel=`echo $line | awk '{print $2}'`
-    python $script_location/install_cybershake.py $cybershake_root $cybershake_cfg $fault --n_rel ${n_rel//[nkr]} --version $version
```
[the remark this answers is not in the document]
Comment (ttgenius) [reply scripts/cybershake/install_cybershake.sh]: Don't need pass $version any more as  $version is in cybershake_config.json now

File: estimation/estimate_wct.py (hunk 2 above)
Comment (ttgenius) [reply estimation/estimate_wct.py]: updated

**Comment by ttgenius** [comment]
updated

**Pull request merged**